src/lib/viewers/BaseViewer.js

```diff
@@ -1098,14 +1104,14 @@ class BaseViewer extends EventEmitter {
 
         this.annotationControls.setMode(mode);
 
-        if (this.options.enableAnnotationsDiscoverability && this.containerEl) {
-            switch (mode) {
-                case AnnotationMode.REGION:
-                    this.containerEl.classList.add(CLASS_ANNOTATIONS_CREATE_REGION);
-                    break;
-                default:
-                    this.containerEl.classList.remove(CLASS_ANNOTATIONS_CREATE_REGION);
-                    break;
+        if (this.containerEl) {
```

Comment:
I removed the `enableAnnotationsDiscoveribility` clause because these classes can be applied with or without that feature. This allows explicit creation modes to disable existing annotations interactivity irrespective of discoverability